### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by LilithNixx** [community/community#69566] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can someone tell me how to get in contact with people that works here? I've been trying to push my proyect for 2 days by now and this message appears: 
fatal: unable to access 'https://githug.com/LilithNixx/ecommerce.git/': Failed to connect to githug.com port 443 after 225016 ms: Timed ouce.git/': Failed to connect to githug.com port 443 after 225016 ms: Timed out

**Comment by gilgongo** [comment]
Did mean "githuB.com"?
